docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+2. Provision the Redis service through the Service Catalog and use it as an alternative database for the microservice.
+3. Connect am external mock application as an addon to Kyma. We will use it to send events to the microservice with order delivery details.
+4. Create a Function with the logic similar to that of the microservice. You will see how it can also be triggered by order events from the mock application, and save the order details in the external storage.
+
+> **CAUTION:** These guides refer to a sample `orders-service` application deployed on Kyma as a `microservice`. This way it can be easily distinguished from the external Commerce mock that represents an external monolithic application connected to Kyma. In Kyma docs, they are referred to as `application` and `Application` respectively.
+
+All guides, whenever possible, demonstrate the steps from both kubectl and UI.
+
+## Prerequisites
+
+1. Install:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (1.16 or greater)
+- [curl](https://github.com/curl/curl)
+- [Kyma](/root/kyma/#installation-install-kyma-on-a-cluster) on a cluster of your choice.
+
+2. Download the `kubeconfig` file with the cluster configuration:
+
+a. Access the Console UI of your Kyma cluster.
+
+b. Click the user icon in the upper right corner.
+
+c. Select **Get Kubeconfig** from the drop-down menu to download the configuration file to a selected location on your machine.
+
+d. Open a terminal window.
+
+e. Export the **KUBECONFIG** environment variable to point to the downloaded `kubeconfig`. Run this command:
+
+   ```bash
+   export KUBECONFIG={KUBECONFIG_FILE_PATH}
+   ```
+
+   >**NOTE:** Drag and drop the `kubeconfig` file in the terminal to easily add the path of the file to the `export KUBECONFIG` command you run.
+
+f. Run `kubectl cluster-info` to check if you are connected to the correct cluster.
+
+## Main actors
+
+Let's introduce the main actors that will lead us through the guides. These are our in-house examples, mock applications, and experimental services:
+
+- [`orders-service`](https://github.com/kyma-project/examples/tree/master/orders-service) is a sample microservice written in Go. It can expose HTTP endpoints that can be used to create and read basic order JSON entities. The service can run with either an in-memory database that is enabled by default or an external Redis database. You will see on its example how you can use Kyma to deploy your own microservice, expose it on HTTP endpoints to make it available for external services, and then bind it to an external database service (Redis).
+
+- [`orders-function`](https://github.com/kyma-project/examples/tree/master/orders-service/deployment/function.yaml) that is an equivalent of `orders-service` with the ability to retrieve order records and save data in it. Like the microservice, the Function can run with either an in-memory database or a Redis instance.
```

Comment:
```suggestion
- [`orders-function`](https://github.com/kyma-project/examples/tree/master/orders-service/deployment/function.yaml) that is an equivalent of `orders-service` with the ability to retrieve order records and save data to them. Like the microservice, the Function can run with either an in-memory database or a Redis instance.
```
`to retrieve order records and save data in it` -> `to retrieve order records and save data to them`, i.e. save data to the retrieved records? 
Otherwise, it's not clear to me what `it` refers to, where it saves the data.

Reply:
`it` = `storage`. I rephrased that sentence :)